src/ray/core_worker/object_interface.cc

```diff
@@ -35,32 +55,26 @@ Status CoreWorkerObjectInterface::Get(const std::vector<ObjectID> &ids,
                                       std::vector<std::shared_ptr<RayObject>> *results) {
   (*results).resize(ids.size(), nullptr);
 
-  // Divide the object ids into two groups: direct call return objects and the rest,
-  // and de-duplicate for each group.
-  std::unordered_set<ObjectID> direct_call_return_ids;
-  std::unordered_set<ObjectID> other_ids;
-  for (const auto &object_id : ids) {
-    if (object_id.IsReturnObject() &&
-        object_id.GetTransportType() ==
-            static_cast<int>(TaskTransportType::DIRECT_ACTOR)) {
-      direct_call_return_ids.insert(object_id);
-    } else {
-      other_ids.insert(object_id);
-    }
-  }
+  // There can be a few cases here:
```

Comment:
This should be moved to the header comment for `GetObjectIdsPerStoreProvider()`

Reply:
yes this makes sense. thanks.